templates/named.conf.local.forward.j2

```diff
@@ -0,0 +1,17 @@
+## {{ ansible_managed }}
+{% for fwd_zone in bind_config_forward_zones %}
+######## {{ fwd_zone.name }} ({{ fwd_zone.zones|count }} zones)
+{% for zone in fwd_zone.zones %}
+zone "{{ zone }}" {
+    type forward;
+{% if fwd_zone.forward is defined %}
+    forward {{ fwd_zone.forward }}; 
```

Comment:
while `forward only` is almost certainly always what you want, i'm letting it be specified in the playbook.  maybe the template should default to that if `fwd_zone.forward` isn't specified?